README.md

```diff
@@ -14,28 +34,48 @@ nvm use
 
 and it will install the correct version of nodejs for you.
 
-Now install the necessary packages using `yarn`:
+3. Now install the necessary packages using `yarn`:
 
 ```
 yarn
 ```
 
-## Getting Started
+## Implementing the Use Cases
+
+Here are the steps for implementing the use case demos:
+
+- [Use Case 1: Early Access](/docs/early-access)
```

Comment:
This URL does not work.  I think you need to add .md.

Reply:
Fixed!